apps/vr-tests-web-components/src/stories/slider/slider.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28873&commitId=713b730fdf58d37ff24e9c47a1aadb9edd84e5f5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Default - Dark Mode.normal.chromium.png|0| Added|
|Slider.Default - Dark Mode.rightArrow.chromium.png|0| Added|
|Slider.Default - RTL.normal.chromium.png|0| Added|
|Slider.Default - RTL.rightArrow.chromium.png|0| Added|
|Slider.Default.normal.chromium.png|0| Added|
|Slider.Default.rightArrow.chromium.png|0| Added|
|Slider.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Slider.Disabled - Dark Mode.rightArrow.chromium.png|0| Added|
|Slider.Disabled.normal.chromium.png|0| Added|
|Slider.Disabled.rightArrow.chromium.png|0| Added|
|Slider.Size Medium.normal.chromium.png|0| Added|
|Slider.Size Medium.rightArrow.chromium.png|0| Added|
|Slider.Size Small.normal.chromium.png|0| Added|
|Slider.Size Small.rightArrow.chromium.png|0| Added|
|Slider.Slider Steps Vertical.normal.chromium.png|0| Added|
|Slider.Slider Steps Vertical.rightArrow.chromium.png|0| Added|
|Slider.Slider Steps.normal.chromium.png|0| Added|
|Slider.Slider Steps.rightArrow.chromium.png|0| Added|
|Slider.Vertical.normal.chromium.png|0| Added|
|Slider.Vertical.rightArrow.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>